providers/universal-provider/src/providers/solana.ts

```diff
@@ -26,6 +32,25 @@ class SolanaProvider implements IProvider {
     this.namespace = Object.assign(this.namespace, namespace);
   }
 
+  public requestAccounts(): string[] {
```

Comment:
Intentional that the Solana provider is being updated in this branch/PR too?

Reply:
It's intentional as I've refactored the `requestAccounts` fx. Before, it was requesting by sending request with `eth_getAccounts` which doesn't work outside of evm